scipy/linalg/flapack.pyf.src

```diff
@@ -2788,6 +2788,100 @@ subroutine <prefix2>sbevx(ab,ldab,compute_v,range,lower,n,kd,q,ldq,vl,vu,il,iu,a
 end subroutine <prefix2>sbevx
 
 
+subroutine <prefix2>stebz(d,e,range,vl,vu,il,iu,tol,order,n,work,iwork,m,nsplit,w,iblock,isplit,info)
+  ! computes all or selected eigenvalues of a real, symmetric tridiagonal
+  ! matrix.
+
+  callstatement (*f2py_func)((range>0?(range==1?"V":"I"):"A"),order,&n,&vl,&vu,&il,&iu,&tol,d,e,&m,&nsplit,w,iblock,isplit,work,iwork,&info)
+  callprotoargument char*,char*,int*,<ctype2>*,<ctype2>*,int*,int*,<ctype2>*,<ctype2>*,<ctype2>*,int*,int*,<ctype2>*,int*,int*,int*,int*,int*
+
+  <ftype2> dimension(n),intent(in) :: d
+  <ftype2> dimension(n-1),intent(in) :: e
```

Comment:
This will fail if n==0, so one could as well add a check `check(n>0)` somewhere and replace all `MAX(1,k*n)` expressions by `k*n`.

Reply:
It also needs `depend(n)`